wpilibc/src/main/native/cpp/SensorUtil.cpp

```diff
@@ -46,3 +40,23 @@ bool SensorUtil::CheckAnalogInputChannel(int channel) {
 bool SensorUtil::CheckAnalogOutputChannel(int channel) {
   return HAL_CheckAnalogOutputChannel(channel);
 }
+
+int SensorUtil::DigitalChannels() {
```

Comment:
Probably should be prefixed with Get since other getters in this file do so? (GetDefaultREVPHModule, etc)